ext/session/mod_user.c

```diff
@@ -45,42 +45,39 @@ static void ps_call_handler(zval *func, int argc, zval *argv, zval *retval)
 	}
 }
 
-#define STDVARS								\
-	zval retval;							\
-	zend_result ret = FAILURE
-
 #define PSF(a) PS(mod_user_names).ps_##a
 
-#define FINISH \
-	if (Z_TYPE(retval) != IS_UNDEF) { \
-		if (Z_TYPE(retval) == IS_TRUE) { \
+#define FINISH(return_value) \
```

Comment:
It's nice to get rid of the `STDVARS`/`FINISH` magic, but can't we turn this `FINISH` macro into a function?

Reply:
Good question, I'll have a look

Reply:
Done, although it is quite a strange macro...